src/core_read.cpp

```diff
@@ -30,9 +30,9 @@ CScript ParseScript(std::string s)
 
     if (mapOpNames.empty())
     {
-        for (int op = 0; op <= OP_NOP10; op++)
+        for (int op = 0; op <= FIRST_UNDEFINED_OP_VALUE; op++)
```

Comment:
Why <= rather than <?  This changes the semantics.

Reply:
Yes, I'm not sure how this happened. In https://reviews.bitcoinabc.org/D1141 it is "<".

Reply:
Good eye @dgenr8. 

That’s a bad merge by me. ABC reformatted all the files so cherry picking involves a lot of manual merging.